service/internal/graph/connector.go

```diff
@@ -80,40 +79,28 @@ func (n *connectorNode) buildTraces(
 	}
 	next := connector.NewTracesRouter(consumers)
 
-	switch n.exprPipelineType {
-	case pipeline.SignalTraces:
+	if n.exprPipelineType == pipeline.SignalTraces {
```

Comment:
Can we include this in the switch bellow, will simplify reading.

Same everywhere.

Reply:
Done